eclair.py

```diff
@@ -114,7 +114,7 @@ class EclairNode(object):
 
     displayName = 'eclair'
 
-    def __init__(self, lightning_dir, lightning_port, btc, executor=None,
+    def __init__(self, lightning_dir, lightning_port, btc, electrumx, executor=None,
```

Comment:
Notice that fixtures are instantiated on demand, so adding it as an argument here will start an `electrumx` instance on every test involving `eclair`. Since `eclair` doesn't use electrum this is not necessary and will slow down testing.